src/lib/table/table.md

```diff
@@ -45,3 +45,9 @@ an update via the table's data source.
 In the near future, we will provide a simplified version of the data-table with an easy-to-use
 interface, material styling, array input, and more out-of-the-box features (sorting, pagination,
 and selection).
+
+### Accessibility
+Tables without text or labels should be given a meaningful label via `aria-label` or
+`aria-labelledby`.
+
+Table's default role is 'grid', and it can be changed through `role` attribute.
```

Comment:
We should mention that it can be changed to `treegrid` specifically, since that's the only other role that makes sense (being that it contains rows and gridcells)